### PR TITLE
Readme update to add PIC metric.

### PR DESCRIPTION
**Pull request by gkapish**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,16 @@
-# Saliency Methods
+# Updates
```
Comment (tolga-b) [inline README.md:1]: Updates as the main heading doesn't look right maybe rename this to "Saliency Library" and just list updates on top or have a subtitle (## Updates)?

File: README.md [hunk 2]
```diff
@@ -27,6 +31,16 @@ This repository contains code for the following saliency techniques:
 This list is by no means comprehensive. We are accepting pull requests to add
 new methods!
 
+## Evaluation of Saliency Methods
+
+The repository provides an implementation of Performance Information Curve (PIC) -
+a human independent metric for evaluating quality of saliency methods
```
Comment (tolga-b) [inline README.md:37]: nit: the quality

**Review by tolga-b — APPROVED** [review]
Thank you Andrei, minor comments.

File: README.md (hunk 1 above)
Comment (gkapish) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (gkapish) [reply README.md]: Done.

**Pull request merged**